content/blog/pulumi-2-0-update/index.md

```diff
@@ -0,0 +1,76 @@
+
+<!--more-->
+
+## .NET
+
+We're almost there for [our .NET SDK]({{< relref "/docs/intro/languages/dotnet" >}}). We recently tidied up the API surface area to feel more familiar to .NET developers, added C# getting started guides for [AWS]({{< relref "/docs/get-started/aws?language=csharp" >}}), [Azure]({{< relref "/docs/get-started/azure?language=csharp" >}}), [GCP]({{< relref "/docs/get-started/aws?language=csharp" >}}), and [Kubernetes]({{< relref "/docs/get-started/aws?language=csharp" >}}), and added many new examples. We also closed the gap on all Pulumi features missing from the initial release, including [aliases]({{< relref "/docs/intro/concepts/programming-model#aliases" >}}), [transformations]({{< relref "/docs/intro/concepts/programming-model#transformations" >}}), [StackReference]({{< relref "/docs/intro/concepts/organizing-stacks-projects#inter-stack-dependencies" >}}), and .NET versions of the [Kubernetes](https://github.com/pulumi/pulumi-kubernetes), [Docker](https://github.com/pulumi/pulumi-docker), and [Terraform remote state](https://github.com/pulumi/pulumi-terraform) packages.
+
+The remaining areas to improve for 2.0 include making the API documentation available in C# (details ar below in the "still to come" section), migrating to .NET Core 3.1, and enabling you to more easily test your infrastructure using familiar C# tools.
+
+## Go
+
+We've overhauled [our Go SDK ]({{< relref "/docs/intro/languages/go" >}})to be more idiomatic and easier to use. The changes here are a major improvment as you can see [in this before/after PR](https://github.com/pulumi/examples/commit/b88f385a8951f81fe41cb4f40eb940b074e9263b#diff-07803f7bc3bff97871e6e3eea7287cb8). We previously used `map[string]interface{}` types in many places instead of properly-typed data structures. Not any more &mdash; now you get strong typing for all Go resource APIs across all providers, including AWS, Azure, GCP, and Kubernetes. Examples and documentation have been converted to this new style. [This blog post]({{< relref "/blog/go-sdk-road-to-2" >}}) provides an overview of this new programming model approach.
+
+Although we've had GoDoc support for some time now, we never linked to them! [Now we do.]({{< relref "/docs/reference/pkg?language=golang ">}}) We've also added Go versions to our getting started guides for [AWS]({{< relref "/docs/get-started/aws?language=golang" >}}), [Azure]({{< relref "/docs/get-started/azure?language=golang" >}}), and [GCP]({{< relref "/docs/get-started/gcp?language=golang" >}}), and have many more improvements on the way.
+
+There are a few core features still remaining to flesh out. The biggest one is retiring Dep in favor of Go modules everywhere. We are also adding support for aliases, transformations, and improved API projections, such as supporting union types. Finally, we'll be adding Go support for our Kubernetes, Docker, and Terraform remote state packages.
```

Comment:
`aliases and transformations` -- this is already shipped and documented. 

`supporting union types` -- Is this true? I'm not aware of any plans for this.

Reply:
Great, I started this a while back - you all are moving fast!

Isn't https://github.com/pulumi/pulumi/issues/3803 about union types? It seems to be marked 2.0, although I'll admit it's light on details so I might be misunderstanding.

Reply:
@lukehoban @pgavlin thoughts here? I'm not familiar with this particular item.